lib/ecto/migration/runner.ex

```diff
@@ -208,15 +208,15 @@ defmodule Ecto.Migration.Runner do
     end
   end
 
-  defp repo_and_direction_and_level do
-    Agent.get(runner(), fn %{repo: repo, direction: direction, level: level} ->
-      {repo, direction, level}
+  defp runner_config do
+    Agent.get(runner(), fn %{repo: repo, direction: direction, log: log} ->
+      {repo, direction, log}
     end)
   end
 
-  defp log_and_execute_ddl(repo, level, command) do
+  defp log_and_execute_ddl(repo, [level: level, sql: sql], command) do
```

Comment:
This should either be a tuple or a map. We should not use lists and then pattern match on them with specific ordering because this means the keyword list `[sql: sql, level: level]` would not match and that would be counter-intuitive.